src/_css/sharedStyling.scss

```diff
@@ -217,7 +217,8 @@ input,
 }
 
 #page-limit {
-  width: 45%;
+  width: 75%;
```

Comment:
I hope you have confirmed this won't break any other existing UI

Reply:
Yes. The `page-limit` id is only being used on the `DropdownComponent`